### PR TITLE
feat: added test for length of logs in operation table

### PR DESCRIPTION
**Pull request by SteBaum**
#### Which issue(s) this PR fixes

Fixes None

#### Additional comments

Goes with  `fix: set byte length for operation.logs column` #589 as a test which checks if the `logs` column in the table `operation` can handle a large amount of bytes on each databases. The amount of characters has been set to `10 000 000`. Before the PR #589 the test failed on `MySQL` and `MariaDB` and with the PR it succeeds on these databases.

#### Agreements

- [x] I hereby declare this contribution to be licensed under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0).
- [x] I hereby agree to grant [TOSIT](https://www.tosit.io/) a copyright license to use my contributions.

### CODE REVIEW

**Comment by SteBaum** [comment]
Just added the command for the extra test in the documentation with the last force push

File: poetry.lock
Comment (rpignolet) [inline poetry.lock:1]: Did you use `poetry lock --no-update` to update this file ? You add one dependency and there are 14 dependencies updated.

Comment (SteBaum) [reply poetry.lock:1]: No I will correct this

Comment (SteBaum) [reply poetry.lock:1]: corrected with force push

File: pyproject.toml [hunk 1]
```diff
@@ -29,6 +29,7 @@ pydantic = "^2.6.0"
 psycopg2 = { version = "^2.9.9", optional = true }
 psycopg2-binary = { version = "^2.9.9", optional = true }
 pymysql = { version = "^1.1.0", optional = true }
+python-lorem = "^1.3.0.post1"
```
Comment (rpignolet) [inline pyproject.toml:32]: This is a dependency for running test so it should be in `tool.poetry.group.dev.dependencies` section.

**Review by rpignolet — CHANGES_REQUESTED** [review]

File: pyproject.toml (hunk 1 above)
Comment (SteBaum) [reply pyproject.toml]: done

**Review by rpignolet — APPROVED** [review]

**Review by PaulFarault — APPROVED** [review]

**Pull request merged**